File: src/main/java/ru/vaschenko/ParkPoint/dto/response/SubscriptionResponseDto.java
package ru.vaschenko.ParkPoint.dto.response;

import ru.vaschenko.ParkPoint.dto.ParkingSpaceDto;

import java.time.LocalTime;
import java.util.List;

public record SubscriptionResponseDto(
        ParkingSpaceDto parkingSpace,
        List<Integer> dayOfWeak,
        LocalTime startTime,
        LocalTime endTime
) {}

File: src/dto/SubscriptionResponseDto.js
export class SubscriptionResponseDto {
    constructor(parkingSpace, dayOfWeek, startTime, endTime) {
      this.parkingSpace = parkingSpace;
      this.dayOfWeek = dayOfWeek; // Массив с днями недели [0-6]
      this.startTime = startTime;
      this.endTime = endTime;
    }
  }